docs/src/ComponentsPage.js

```diff
@@ -57,6 +60,28 @@ const ComponentsPage = React.createClass({
             <div className='row'>
               <div className='col-md-9' role='main'>
 
+                {/* Custom styles */}
+                <div className='bs-docs-section'>
+                  <h1 className='page-header'><Anchor id='custom-styles'>Custom styles</Anchor></h1>
+                  <h2><Anchor id='custom-styles-with-warnings'>With no validation.</Anchor></h2>
+                  <p>If you need to use your own custom style class, you can set it via <code>bsStyle</code> property as well as predefined ones. Like that</p>
+                  <div className="highlight">
+                    <CodeExample
+                      mode="javascript"
+                      codeText={
+`<Button bsStyle='myCustom'>My button</Button>
+<Button bsStyle='mybook'>MyBook button</Button>`
+                      }
+                    />
+                  </div>
+                  <p>But it will warn about invalid <code>bsStyle</code> property</p>
+                  <Alert bsStyle='warning'>
+                    <Glyphicon glyph='warning-sign' /> Warning: Failed propType: Invalid prop `bsStyle` of value `myCustom` supplied to `Button`, expected one of ...
+                  </Alert>
+
+                  <h2><Anchor id='custom-styles-with-validation'>With validation</Anchor></h2>
+                  <p>WIP: There is undocumented api that allows adding custom style classes with validation, but it will be changed.</p>
```

Comment:
since the docs aren't finished yet maybe leave them out for now? Otherwise we will need to finish the whole thing before the next release